dotCMS/src/main/java/com/dotcms/ai/service/OpenAIService.java

```diff
@@ -0,0 +1,9 @@
+package com.dotcms.ai.service;
+
+import com.dotmarketing.util.json.JSONObject;
+
+public interface OpenAIService {
+
+    JSONObject getSupportedModels();
```

Comment:
we should stop using JSONObject, use Map instead

Reply:
Add doc

Reply:
Doc added